source/features/comments-time-machine-links.tsx

```diff
@@ -9,10 +9,10 @@ function init() {
 
 	for (const comment of comments) {
 		const timestampEl = select('relative-time', comment);
-		const timestamp = timestampEl.attributes.datetime.value;
+		const timestamp = timestampEl.attributes['datetime'].value; // eslint-disable-line dot-notation
 		const href = `/${getRepoURL()}/tree/HEAD@{${timestamp}}`;
 
-		timestampEl.parentNode.after(
+		(timestampEl.parentNode as HTMLElement).after(
```

Comment:
In this case we're casting because the TypeScript types in I believe lib.dom.d.ts made parent the following `readonly parentNode: Node & ParentNode | null;`. You could use `parentElement` and then it would work as it's type is `readonly parentElement: HTMLElement | null;`, but I did not want to touch the existing code (aside from types). If you're OK with it being `parentElement`, I'll make the change. It's in a few other places.

Reply:
Sounds good. I feel silly for not knowing there was a time element,  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tim. Will cast accordingly.

Reply:
Yeah `parentElement` is the most appropriate

Reply:
Done.